src/test/appdistro/client.spec.ts

```diff
@@ -200,4 +205,98 @@ describe("distribution", () => {
       });
     });
   });
+
+  describe("createGroup", () => {
+    it("should throw error if request fails", async () => {
+      nock(appDistributionOrigin)
+        .post(`/v1/${projectName}/groups`)
+        .reply(400, { error: { status: "FAILED_PRECONDITION" } });
+      await expect(appDistributionClient.createGroup(projectName, "My Group")).to.be.rejectedWith(
+        FirebaseError,
+        "Failed to create group"
+      );
+      expect(nock.isDone()).to.be.true;
+    });
+
+    const mockResponse: Group = { name: groupName, displayName: "My Group" };
```

Comment:
nit: Put this at the top of the describe block for easier findability

Reply:
I had followed the pattern established by the test for `deleteTesters`, but I now moved that constant as well.